lib/pbench/test/functional/agent/cli/commands/tools/test_tools_list.py

```diff
@@ -54,15 +116,15 @@ def test_no_args(self, tool, agent_config):
         out, err, exitcode = pytest.helpers.capture(command)
         assert TRACEBACK not in err
         assert EMPTY == err
-        assert b"default: testhost.example.com ['perf']" in out
+        assert b"group: default; host: testhost.example.com; tools: perf\n" in out
```

Comment:
Shouldn't `mpstat` be here, too?

Reply:
Ditto line 150, et al.

Reply:
Why should `mpstat` be here (or on l.150)? The `tool` fixture just creates a `perf` tool.

Reply:
I must have been confused about which fixture was used to set up the tools list.